beer_and_tell/2016-09-September.md

```diff
@@ -0,0 +1,80 @@
+
+- 499 BCE: Histiaeus of Miletus shaves the heads of messengers, tattoos messages on their scalps, and sends them after their hair has grown back to hide the message.
+- ~100 AD: Milk of tithymalus plant is used as invisible ink, activated by heat.
+- ~700 BCE: [Scytale][]
+- 49 BC: [Caesar cipher][]
+- 1553 AD: [Vigenère cipher][]
+
+[groovecoder]: https://mozillians.org/en-US/u/groovecoder/
+[codebook]: https://en.wikipedia.org/wiki/The_Code_Book
+[Scytale]: https://en.wikipedia.org/wiki/Scytale
+[Caesar cipher]: https://en.wikipedia.org/wiki/Scytale
+[Vigenère cipher]: https://en.wikipedia.org/wiki/Vigen%C3%A8re_cipher
+
+## bensternthal: Home Monitoring & Weather Tracking
+[bensternthal][] was up next, and he shared his work building a dashboard with weather and temperature information from his house. Ben built several Node.js-based applications that collect data from his home weather station, from his [Nest][] thermostat, and from [Weather Underground][] and send all the data to an [InfluxDB][] store. The dashboard itself uses [Grafana][] to plot the data, and all of these servers are run using [Docker][].
+
+The repositories for the Node.js applications and the Docker configuration are available on Github:
+
+- [grafana-influxdb-docker](https://github.com/bensternthal/grafana-influxdb-docker)
+- [wunderground-influxdb](https://github.com/bensternthal/wunderground-influxdb)
+- [nest-influxdb](https://github.com/bensternthal/nest-influxdb)
+- [tfws-influxdb](https://github.com/bensternthal/tfws-influxdb)
+
+[bensternthal]: https://mozillians.org/en-US/u/bensternthal/
+[Nest]: https://nest.com/
+[Weather Underground]: https://www.wunderground.com/
+[InfluxDB]: https://www.influxdata.com/
+[Grafana]: http://grafana.org/
+[Docker]: https://www.docker.com/
+
+## craigcook: ByeHolly
+Next was [craigcook][], who shared a [virtual yearbook page][] that he made as a tribute to former-Mozillian habber, who recently left. The page shows several photos that are clipped at the edges to look curved like an old television screen. This is done in CSS using [clip-path][] with an [SVG][]-based path for clipping. The SVG used is also defined using proportional units, which allows it to warp and distort correctly for different image sizes, as seen by the variety of images it is used on in the page.
+
+[craigcook]: https://mozillians.org/en-US/u/craigcook/
+[virtual yearbook page]: http://craigcook.github.io/etc/byeholly/
+[clip-path]: https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path
+[SVG]: https://developer.mozilla.org/en-US/docs/Web/SVG
+
+## peterbe: react-buggy
+[peterbe][] told us about [react-buggy][], a client for viewing Github issues implemented in [React][]. It is a rewrite of [buggy][], a similar client peterbe wrote for Bugzilla bugs. Issues are persisted in [IndexedDB][] so that the app can function offline. The client also uses [elasticlunr.js][] to provide full-text search on issue titles and comments.
```

Comment:
Also, instead of linking to `IndexedDB` can you change that to [Lovefield](https://github.com/google/lovefield) and add in a bracket after "(a wrapper for IndexedDB)".